data_steward/analytics/cdr_ops/rdr_export_qc.py

```diff
@@ -425,3 +425,20 @@
 WHERE primary.research_id <> rdr.research_id
 '''
 pd.read_gbq(query, dialect='standard')
+
+# Checks for basics survey module
+# Participants with data in other survey modules must also have data from the basics survey module.
+# This check identifies survey responses associated with participants that do not have any responses
+# associated with the basics survey module.
+# In ideal circumstances, this query will not return any results.
+
+query = f'''SELECT DISTINCT person_id FROM `{project_id}.{new_rdr}.person` 
+WHERE NOT EXISTS (
```

Comment:
```suggestion
WHERE person_id NOT IN (
```
I think this is the correct way to find persons who are not selected from the subquery. `WHERE NOT EXISTS` will always return no rows if at least 1 row is returned by the subquery.

Reply:
Nice catch! I'll make that change and repush!